packages/core/admin/server/routes/admin.js

```diff
@@ -7,6 +7,20 @@ module.exports = [
     handler: 'admin.init',
     config: { auth: false },
   },
+  {
+    method: 'GET',
+    path: '/project-settings',
+    handler: 'admin.getProjectSettings',
+    config: {
+      policies: [
+        'admin::isAuthenticatedAdmin',
+        {
+          name: 'admin::hasPermissions',
+          config: { actions: ['admin::project-settings.update'] },
```

Comment:
Shouldn't it be `config: { actions: ['admin::project-settings.read'] },` (`read` instead of `update`) here?

Reply:
You are right

Reply:
I added a dedicated read permission